tests/conftest.py

```diff
@@ -31,6 +31,91 @@
 )
 
 
+CHECK_AGG_DF = pd.DataFrame([
```

Comment:
This looks to me like a much larger dataframe than what would be needed to run the unit tests...?

Reply:
ye so I wasn't completely sure how best to do this. Basically I wanted a fake dataset where I had a variable with components, a variable with no components, a variable like CO2 which has data at the world level which needs to go in the world total but isn't in the regions and at least 2 regions (to check the aggregate against the world) so seemed like I need this...